docs/development.md

```diff
@@ -138,15 +179,23 @@ Test can be run using the configurations available under the green `Run` button
 Additionally, configurations that run the tests with `GodMode` turned on are also provided.
 More info on this can be found [here](godmode.md).
 
+### Running the test suite in an IDE
+
+When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab until they pass.
+
+#### Eclipse
+
+Run tests using the configurations available under the green `Run` button on the Eclipse toolbar.
+
 Sometimes, Eclipse does not show these options immediately after you set up the project. "Refreshing" the project should fix that.
 
-New developers should run `All tests` and have all of them passing at least once on their local environments.
+#### IntelliJ
 
-When running the test cases, if a few cases fail (this can happen due to timing issues), run the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse until they pass.
+Run tests using the configurations available under `Run → Play`.
```

Comment:
There is no `Play` in Windows. Did you mean the triangle symbol? And there are no test configurations to run anyway.